lib/multi.c

```diff
@@ -3043,3 +3051,11 @@ void Curl_multi_dump(struct Curl_multi *multi)
   }
 }
 #endif
+
+size_t Curl_multi_max_concurrent_streams
+(struct Curl_multi *multi)
+{
+  return multi ?
+    (multi->max_concurrent_streams?
+    multi->max_concurrent_streams:100) : 0;
```

Comment:
these lines are too broken up they can be longer up to 79

Reply:
Thanks, updated.